mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -85,6 +85,31 @@ func resourceBigtableInstance() *schema.Resource {
 							Computed:    true,
 							Description: `Describes the Cloud KMS encryption key that will be used to protect the destination Bigtable cluster. The requirements for this key are: 1) The Cloud Bigtable service account associated with the project that contains this cluster must be granted the cloudkms.cryptoKeyEncrypterDecrypter role on the CMEK key. 2) Only regional keys can be used and the region of the CMEK key must match the region of the cluster. 3) All clusters within an instance must use the same CMEK key. Values are of the form projects/{project}/locations/{location}/keyRings/{keyring}/cryptoKeys/{key}`,
 						},
+						"autoscaling_config": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							MaxItems:    1,
+							Description: "A list of Autoscaling configurations. Only one element is used and allowed.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"min_nodes": {
+										Type:     schema.TypeInt,
+										Required: true,
+										Description: `The minimum number of nodes for autoscaling.`,
```

Comment:
nit: "The minimum number of nodes to scale down to"?

Similarly, "Maximum number of nodes to scale up to."